.github/workflows/test.yaml

```diff
@@ -0,0 +1,52 @@
+name: test
+
+on:
+  pull_request:
+    branches: ['*']
+
+jobs:
+  test:
+
+    name: Run tests
+
+    runs-on: ubuntu-latest
```

Comment:
is it possible to run this on latest debian as well?

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

No. Could run it on a mac, or windows, but I don't think it really matters? It's the same architecture anyway (?)

Reply:
yea it doesn't really matter i guess